File: Source/Fantasy.Jobs/InvalidJobStartInfoException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Fantasy.Jobs
{
    [Serializable]
    public class InvalidJobStartInfoException : Exception 
    {
        public InvalidJobStartInfoException(string message, Exception innerException)
            : base(message, innerException)
        {

        }

        public InvalidJobStartInfoException(string message)
            : base(message)
        {

        }
    }
}

File: Fantasy.Jobs/src/fantasy/jobs/InvalidJobStartInfoException.java
package fantasy.jobs;

public class InvalidJobStartInfoException extends RuntimeException implements Serializable
{
	public InvalidJobStartInfoException(String message, RuntimeException innerException)
	{
		super(message, innerException);

	}

	public InvalidJobStartInfoException(String message)
	{
		super(message);

	}
}